### PR TITLE
Markdown stdlib

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

**Comment by ararslan** [comment]
Gotta love the CI situation. Red across the board, all unrelated.

File: README.md [hunk 1]
```diff
@@ -141,6 +141,9 @@ Currently, the `@compat` macro supports the following syntaxes:
 * `using Compat.UUIDs` is provided on versions older than 0.7, where this library is
   not yet part of the standard library ([#25819]).
 
+* `using Compat.Markdown` is provided on versions older than 0.7, where this library is
+  not yet part of the standard library ([#25738]).
```
Comment (omus) [inline README.md]: Needs the full link added at the bottom of the page. Currently is a broken link.

Comment (ararslan) [reply README.md]: We need a script here that does what `news-update.jl` or whatever it's called does in Base.

Comment (omus) [reply README.md]: That would be good. It would also be good to add this as a CI check.

Comment (fredrikekre) [reply README.md]: ran the script in #493

**Pull request merged**